### PR TITLE
update iosxr readme

### PR DESCRIPTION
**Pull request by samccann**
##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

documentation
##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+
+```
+
+Alternately, you can call modules by their short name if you list the `cisco.iosxr` collection in the playbook's `collections`, as follows:
+
+```yaml
+---
+- hosts: iosxr01
+  gather_facts: false
+  connection: network_cli
```
Comment (rohitthakur2590) [inline README.md:65]: when FQCN is used in task name 'collections' keyword is not required.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+              - name: acl_1
+                aces:
+                  - sequence: 23
+        state: deleted
+```
+
+
+### See Also:
+
+* [Cisco IOSXR Platform Options](https://docs.ansible.com/ansible/latest/network/user_guide/platform_iosxr.html)
```
Comment (rohitthakur2590) [inline README.md:85]: Are we updating p?platform guide w.r.t FQCN?

Comment (samccann) [reply README.md:85]: tracking this as https://github.com/ansible/ansible/issues/69374

File: README.md (hunk 1 above)
Comment (samccann) [reply README.md:65]: not sure what this comment refers to here?  This example is specifically that - includes `collections` keyword and does not use FQCN.

Comment (rohitthakur2590) [reply README.md:65]: Oh yeah, you are right, I made a mistake here.

**Pull request merged**